services/app-api/forms/mcpar.json

```
@@ -4425,7 +4417,7 @@
                 "validation": "number",
                 "props": {
                   "label": "D1.XIII.5 Percentage of standard PA requests that were denied",
-                  "hint": "Enter the percentage of the total standard PA requests, as reported in D1.XIII.1, that were denied. If you choose not to respond prior to June 2026, enter \"NR\" for not reporting.",
+                  "hint": "Enter the percentage of the total standard PA requests, as reported in D1.XIII.1, that were denied. If you choose not to respond prior to June 2026, enter “NR“ for not reporting.",
```

fix double open quote
(same for questions D1.XIII.6-13)
```suggestion
                  "hint": "Enter the percentage of the total standard PA requests, as reported in D1.XIII.1, that were denied. If you choose not to respond prior to June 2026, enter “NR” for not reporting.",
```